### PR TITLE
Filip/rules/chamberlin courant brute force

### PR DESCRIPTION
**Pull request by Wolokin**

### CODE REVIEW

**Comment by Wolokin** [comment]
[CC-rule implementation: Brute-force](https://trello.com/c/MzprsPqO/39-cc-rule-implementation-brute-force)

File: votepy/chamberlin-courant.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    Returns:
+        list[int]: List of chosen candidates
+    """
+    def scoring_function(committee: Iterable[int], voting: OrdinalElection, number_of_scored_candidates: int):
+        committee = set(committee)
+        score = 0
+        for vote in voting:
+            for i, candidate in enumerate(vote):
+                if candidate in committee:
+                    score += max(number_of_scored_candidates - i, 0)   # should it be 0 when outside the number of scored candidates?
```
Comment (msluszniak) [inline votepy/chamberlin-courant.py]: Yes, it should
```suggestion
                    score += max(number_of_scored_candidates - i, 0)
```

**Review by msluszniak — COMMENTED** [review]
Everything seems to be ok except the casting in generic_brute_force.py

File: votepy/generic_brute_force.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Returns:
+        list[int]: The winning committee
+    """
+    best_committee, best_score = None, -1
+    for committee in combinations(range(voting.ballot_size), size_of_committee):
+        score = scoring_function(committee, voting)
+        if score > best_score:
+            best_score = score
+            best_committee = committee
+    return best_committee
```
Comment (msluszniak) [inline votepy/generic_brute_force.py]: combinations return iterable of tuples so at the end you need to cast best_committee to a list.
```suggestion
    return list(best_committee)
```

Comment (Wolokin) [reply votepy/generic_brute_force.py]: Yeah, I've missed that, thanks

**Pull request merged**